tests/versioned/apollo-server/query-obfuscation.test.js

```diff
@@ -9,7 +9,7 @@ const test = require('node:test')
 
 const { afterEach, setupCoreTest } = require('../../lib/test-tools')
 
-const queryObfuscationTests = require('../query-obfuscation-tests')
+const queryObfuscationTests = require('./query-obfuscation-tests')
```

Comment:
i don't see the need for this file. you can prob just move the `query-obfuscation-tests` into here or make that file run the tests.